src/components/SearchResults/UserContact/UserContact.js

```diff
@@ -1,34 +1,37 @@
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faLink,
+  faLocationDot,
+  faBuilding,
+} from "@fortawesome/free-solid-svg-icons";
+import { faTwitter } from "@fortawesome/free-brands-svg-icons";
 import styles from "./UserContact.module.css";
 
-const UserContact = ({
-  contactInfo: { job, location, twitter, website },
-  icons: [locationIcon, twitterIcon, websiteIcon, jobIcon],
-}) => {
+const UserContact = ({ location, twitter_username, blog, company }) => {
   const contactData = [
     {
-      name: location,
-      icon: locationIcon,
+      name: `${location || "Not Available"}`,
```

Comment:
We can remove the string interpolation here as well. Comparison check is enough on its own
```suggestion
      name: location || "Not Available",
```